src/devices/src/virtio/net/device.rs

```diff
@@ -1291,7 +1314,7 @@ pub mod tests {
             ],
         );
         let frame = th.check_rx_deferred_frame(1000);
-        th.rxq.check_used_elem(0, 0, 100);
+        th.rxq.check_used_elem(0, 0, 0);
```

Comment:
So we can either send the entire frame or nothing? Is this intended?

Reply:
Yes, it's intended. We either send the entire frame or we skip the descriptor chain. There's no benefit in sending incomplete invalid frames.